pdp/static/js/crmp_app.js

```diff
@@ -33,6 +33,10 @@ $(document).ready(function () {
     map.composite_filter = '';
     map.getControlsByClass('OpenLayers.Control.DrawFeature')[0].events.register('featureadded', '', pdp.curry(polyChange, map));
 
+    // Don't include the full timeseries download option, some of the
+    // stations don't have start/end dates and will get excluded
+    $('#download-all-time').remove()
```

Comment:
I find this to be a little non-intuitive. Create a bunch of elements and then manually remove one of them? It's a bit awkward. Maybe a better way would be to include a boolean parameter to `getDateRange()` named `omitFullTimeSeriesCheckbox` (or something less verbose) that's False by default?